docs/sql-ref-syntax-ddl-alter-view.md

```diff
@@ -19,4 +19,78 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+The `ALTER VIEW` statement changes various auxiliary properties of a view.
+
+
+#### Rename view
+Rename the existing view. If the view name already exists in the database, an exception is thrown. This operation does 
+support moving the views cross databases. 
+##### Syntax
+{% highlight sql %}
+ALTER VIEW viewIdentifier RENAME TO viewIdentifier
+viewIdentifier:= [db_name.]view_name
+{% endhighlight %}
+
+
+#### Set view properties
+Set one or more properties of an existing view. The properties are the key value pairs. If the properties' keys exist, 
+the values are replaced with the new values. If the properties' keys does not exist, the key value pairs are added into 
+the properties.
+##### Syntax
+{% highlight sql %}
+ALTER VIEW viewIdentifier SET TBLPROPERTIES (key1=val1, key2=val2, ...)
+viewIdentifier:= [db_name.]view_name
+{% endhighlight %}
+
+
+#### Drop view properties
```

Comment:
You may need a blank line after section headings, I forget. It'd be clearer in the markdown anyway